File: DataStructures/LinkedList/SimpleLinkedList3Nodes.java
package DataStructures.LinkedList;
public class SimpleLinkedList3Nodes {

    Node head;   //head of list

    // Linked list node. The inner class is amde static so that main can acces it 

    static class Node {
        int data;
        Node next;
        Node (int d);
        {
            data = d;
            next = null;
        } //Constructor
    }
    
public static void main(String[] args)
{
    LinkedList llist = new SimpleLinkedList3Nodes();
    llist.head = new Node(1);
    Node second = new Node(2);
    Node third = new Node(3);

    llist.head.next = second;
    second.next = third;




    }
    
}

File: DataStructures/LinkedList/SimpleLinkedList3Nodes.cs
using System;

public class LinkedList {
    Node head; //head of list

    public class Node{
        public int data;
        public Node next;
        public Node(int d)
        {
            data = d;
            next = null;
        }
    }

    public static void Main(String[] args)
    {
        LinkedList llist = new LinkedList();

        llist.head = new Node(1);
        Node second = new Node(2);
        Node third = new Node(3);

        llist.head.next = second;

        second.next = third;

    

    }


}